.github/workflows/flink-ci.yml

```diff
@@ -20,25 +20,8 @@
 name: "Flink CI"
 on:
   push:
-    paths-ignore:
-    - '.github/workflows/python-ci.yml'
-    - '.github/workflows/spark-ci.yml'
-    - '.github/workflows/hive-ci.yml'
-    - '.github/workflows/cancel-duplicate-workflow-runs.yml'
-    - '.gitignore'
-    - 'dev/**'
-    - 'mr/**'
-    - 'hive3/**'
-    - 'hive3-orc-bundle/**'
-    - 'hive-runtime/**'
-    - 'spark/**'
-    - 'pig/**'
-    - 'python/**'
-    - 'python_legacy/**'
-    - 'docs/**'
-    - '.gitattributes'
-    - 'README.md'
-    - 'CONTRIBUTING.md'
+    branches:
+    - 'master'
```

Comment:
I suspect that adding
```
    - '*.x'
```
could work to have CI run on release branches. If it might make sense to move to a different naming pattern for release branches (like `release-x.y.z`).

Reply:
For now, I just added `'**'` to catch all, as that's the only thing we use tags for presently anyway.

The tags from the examples use two asterisks, so I went with that: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs

I can try merging this into my own fork and then pushing a tag to ensure it works or we can just merge it and then check in the fork (less work). I'm open either way.